slides.md

```diff
@@ -1427,7 +1427,7 @@ export default async function users(fastify) {
 <div class="dense">
 
 - Let's create an Fastify application using **TypeScript**.
-- We will transpose the application that you did in the [Step 09](#step-9-decorators) to TypeScript
+- We will transpose the application that you did in the [Step 10](/59) to TypeScript
```

Comment:
are you sure that anchors are not supposed to work? (you may be right, I'm just double checking)

Reply:
I didn't manage to make it work locally, I also tried to search about anchors or redirects it in the official Slidev documentation but didn't find any content about it 😕